R/plotreg.R

```diff
@@ -1,394 +1,462 @@
+#'  \code{\link{screenreg}}, \code{\link{texreg}}, and \code{\link{htmlreg}} 
+#'  \code{\link{matrixreg}} and \code{\link{wordreg}} functions.It is 
+#'  possible to display the plots in two ways: using the \code{type = "facet"} 
+#'  option, one forest plot applied to point estimates and confidence intervals 
+#'  will be visualised in case there is only one model. If there is more than 
+#'  one model each one will be plotted next to the other; using the 
+#'  \code{type = "forest"} option, coefficients from one or more models will 
+#'  be grouped together and displayed as a forest plot. 
+#'
+#' 
+#' @param custom.title With this argument, a replacement text for the \code{ggtitle} 
+#'   that provides a title above the diagram can be provided. If an empty character 
+#'   object is provided (\code{custom.note = ""}), the note will be omitted completely.
+#' @param override.pval Replacement list of p-value vectors.
+#' @param ci.level If standard errors are converted to confidence intervals
+#'   (because a model does not natively support CIs), what confidence level 
+#'   should be used for the outer confidence interval? By default, \code{0.95}
+#'   is used (i.e., an alpha value of 0.05).
+#' @param use.se Use one standard error for the inner horizontal bar and two 
+#'   standard errors from the estimate for the outer horizontal bar (instead 
+#'   of confidence intervals). Only available if standard errors can be 
+#'   extracted from the model using the respective \code{\link{extract}} function.
+#' @param type The default option is \code{type = "facet"}. If one model only is 
+#'   specified, it will print one forest plot applied to point estimates and 
+#'   confidence intervals. If more than one model is specified, it will print 
+#'   as many plots as the number of models in a column of plots. Alternatively,
+#'   if \code{type = "forest"} is specified, coefficients from one or more models 
+#'   will be grouped together and displayed as a forest plot.
+#' @param theme The \code{\link{theme}} argument can be used to customomise 
+#'   the non data component of your plot. The default \code{\link{theme}} is
+#'   \code{theme_bw} and it can be substituted by any other theme compatible 
+#'   with \pkg{ggplot2}.
+#' @param signif.light Color of outer confidence intervals for significant model terms.
+#' @param signif.medium Color of inner confidence intervals for significant model terms.
+#' @param signif.dark Color of point estimates and labels for significant model terms.
+#' @param insignif.light Color of outer confidence intervals for insignificant model terms.
+#' @param insignif.medium Color of inner confidence intervals for insignificant model terms.
+#' @param insignif.dark Color of point estimates and labels for insignificant model terms.
+#' 
+#' @return coefficient plots from statistical model output using \pkg{ggplot2}.
+#' 
+#' @inheritParams texreg
+#' @inheritParams matrixreg
+#'
+#'
+#' @author Philip Leifeld
```

Comment:
I think this should be `@author Claudia Zucca, Philip Leifeld` because we both contributed significant code portions to this function.